src/mixins/map-mixin.js

```diff
@@ -521,6 +523,18 @@ export default function mapMixin(
     }
   })
 
+  function getFirstSymbolLayerId () {
+    let firstSymbolId = null
+    const layers = _map.getStyle().layers
+    for (let i = 0; i < layers.length; ++i) {
+      if (layers[i].type === "symbol") {
+        firstSymbolId = layers[i].id
+        break
+      }
+    }
+    return firstSymbolId
```

Comment:
I guess nothing bad happens when firstSymbolId is null? I tried the various styles and it was all working.

Reply:
I _think_ this function will only be called when the basemap has been loaded, so not sure that will return `null`. Did you find a case where it was?

Reply:
No, just curious. I'm barely looking at the logic and seeing that it's technically not impossible that the loop doesn't find any layers[i].type of type "symbol". But it doesn't seem relevant as it works and we can be confident that it finds a match anyway